packages/editor-ui/src/views/TemplatesSearchView.vue

```diff
@@ -10,7 +10,8 @@
 				<div :class="$style.button">
 					<n8n-button
```

Comment:
Would love if we used `<a>` instead of a button when it's a link to specific URL. This way the native browser behaviours would work (ctrl/cmd click etc.) We can still make it look like a button. Eg wrap the button into a `<n8n-link>`

Reply:
Agree. We have to extend our button so it supports more tags than just `<button>` while preserving theme support